### PR TITLE
CDS-96147-Monitored-Service-Template-as-Runtime-Input

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.
https://harness.atlassian.net/browse/CDS-96147

### CODE REVIEW

**Comment by shankaranandh** [comment]
Approved

**Review by SushrutHarness — COMMENTED** [review]
review To be continued

File: docs/continuous-delivery/verify/cv-concepts/cv_templates.md [hunk 1]
```diff
@@ -5,53 +5,75 @@ description: Learn about strategies to template out CV workflow items.
...
+   ![](./static/cv_templates/select-template-runtime.png)
+4. Select **Apply Selected**.
+
+
+## Advanced use cases
+
+A monitored service template allows the following use cases:
 
 * Multiple verify steps with different monitored service templates can be configured to be run in a particular stage. These steps can be then configured to be run based on certain conditions.
 * A monitored service template with user impacting metric (not necessarily satisfied/linked to a single service) can be created and use in verification in addition to a template with service linked metric.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/verify/cv-concepts/cv_templates.md:76]:
```suggestion
* A monitored service template with an user impacting metric (not necessarily satisfied/linked to a single service) can be created and use in verification in addition to a template with service linked metric.
```

File: docs/continuous-delivery/verify/cv-concepts/cv_templates.md [hunk 2]
```diff
@@ -5,53 +5,75 @@ description: Learn about strategies to template out CV workflow items.
...
 
-## Verify Step Template
+![](./static/cv_templates/select-template-fixed-value.png)
 
-Like any other construct of pipeline (step, stage, pipeline); a step template for verify step can also be created. Verify step template can be used in conjunction with other pipeline constructs like pipeline, stage template etc.
+### Select a monitored service template during runtime
 
-It also allows, user to specify whether it wants to use the “default” or “template” option for the monitored service. What it means is that a monitored service template can be used as part of concrete verify step and also with verify step template. If the purpose is to standardize the step configuration then create a step template for verify step. 
+:::important
+This feature is currently behind the feature flag, `CV_MONITORED_SERVICE_TEMPLATIZATION`. To enable a feature flag in your Harness account, contact [Harness Support](mailto:support@harness.io).
```
Comment (SushrutHarness) [inline docs/continuous-delivery/verify/cv-concepts/cv_templates.md:54]: Is specifically the ability to select a template during runtime behind this flag? Should we be more specific what part of templatization is behind a future flag?

**Review by SushrutHarness — APPROVED** [review]
Left a few comments otherwise LGTM!

File: docs/continuous-delivery/verify/cv-concepts/cv_templates.md (hunk 2 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/verify/cv-concepts/cv_templates.md:54]: yes

**Pull request merged**